README.md

```diff
@@ -69,3 +69,92 @@ The dependencies json lists specific rules for the validator to follow. When lis
+* If `scenes[n].action_mapping[m].action_type` is “APPLY_TREATMENT”, `scenes[n].action_mapping[m].character_id` is required
+* If `scenes[n].action_mapping[m].action_type` is “CHECK_ALL_VITALS”, `scenes[n].action_mapping[m].character_id` is required
+* If `scenes[n].action_mapping[m].action_type` is “CHECK_PULSE”, `scenes[n].action_mapping[m].character_id` is required
+* If `scenes[n].action_mapping[m].action_type` is “CHECK_RESPIRATION”, `scenes[n].action_mapping[m].character_id` is required
+* If `scenes[n].action_mapping[m].action_type` is “MOVE_TO_EVAC”, `scenes[n].action_mapping[m].character_id` is required
+* If `scenes[n].action_mapping[m].action_type` is “TAG_CHARACTER”, `scenes[n].action_mapping[m].character_id` is required
+* If `scenes[n].action_mapping[m].action_type` is “TAG_CHARACTER”, `scenes[n].action_mapping[m].parameters.category` is required
+
+#### Conditional Prohibitions
+* If `scenes[].index` is 0, `scenes[].state` should _not_ be provided
+* If `state.characters[n].demographics.military_branch` does not exist, `state.characters[n].demographics.rank` *and* `state.characters[n].demographics.rank_title` should _not_ be provided 
+
+#### Dependency Allowed Valuese
+* If `state.characters[n].vitals.conscious` is “False”, `state.characters[n].vitals.avpu` should be “UNRESPONSIVE” and `state.characters[n].vitals.mental_status` should be “UNRESPONSIVE” 
+
+#### Value Matching
+* `scenes[n].action_mapping.character_id` must be one of the `state.characters.character_id`'s 
+* `scenes[n].tagging.probe_responses[n].character_id` must be one of the `state.characters.character_id`'s 
+* `scenes[n].action_mapping.conditions.character_vitals.character_id `must be one of the `state.characters.character_id`'s 
+* `scenes[n].tagging.reference` must be one of the `scenes[n].index`'s 
+
+#### Other Rules
+* `scenes[n].action_mapping[m].parameters.treatment` must come from `SupplyTypeEnum` 
+* `scenes[n].action_mapping[m].parameters.location` must come from `InjuryLocationEnum` 
+* `scenes[n].action_mapping[m].parameters.category` must come from `CharacterTagEnum` 
+* If there are `N` `scenario.scenes`, then `scenario.scenes[0]` through `N-1` must contain transitions 
+* `scenario.state.characters.demographics.mission_importance` must be consistent with `scenario.state.mission.character_importance `
+    * Every character with `mission_importance` should be an entry in `character_importance`, and vice-versa 
+    * This does not include “normal”, which is the default level of importance. For example, a character may not specify `mission_importance` and `character_importance` may explicitly specify the character with importance “normal”, or a character may specify `mission_importance` with “normal” and `character_importance` may not list that character 
+
+#### Injury/Location Matches
+Injuries are only allowed to have specific locations. Please follow the table to create valid matches.
```

Comment:
@brianpippin   This is totally unvetted and just pretty much what I came up with off the top of my head, with some input from Kaitlyn.  I don't even know for sure that you want this validated.  So please let us know what you think.

Reply:
@brianpippin

Reply:
Yes injuries should only be allowed at certain locations because that is what the Simulation supports so that is fine by me.

Reply:
I guess what I'm saying is that for the most part, I just kind of made up the valid locations for each injury based on what I thought made sense, and didn't cross-reference against the sim except based on what we had documented [here](https://nextcentury.atlassian.net/wiki/spaces/ITM/pages/3030581249/List+of+injuries+and+their+treatments).

But e.g. Broken Bones are new and I'm not sure that Abrasions can go anywhere but on the forehead, so I just think we need to make sure there's nothing loose here or incompatible with the sim.  It doesn't need to hold up this ticket, though.

Reply:
I think we leave as is for what we have in the Sim not what we might add to the sim.  As new injuries are added to the Sim this code should be updated then, but not until the injuries are added or confirmed to be added for an evaluation.  So I think this is all good to go.